libhb/dvd.c

```diff
@@ -242,6 +242,22 @@ static void add_subtitle( hb_list_t * list_subtitle, int position,
             break;
     }
 
+    switch (style)
+    {
+        case HB_VOBSUB_STYLE_4_3:
+            strcat( subtitle->lang, " (4:3)" );
```

Comment:
Do you mean in the text? e.g. [4:3]?  Why would that be "better"?  Parens match the other descriptive items we append to the description.